### PR TITLE
chore: internal-release -> edge for 0.13.0-beta.0

### PR DESCRIPTION
**Pull request by sfoster1**
This is a mergeback to edge for internal release 0.13.0-beta.0

### CODE REVIEW

**Comment by sfoster1** [comment]
`Results.tsx` merge resolution (remove the Z home):
```
@@ -138,11 +138,8 @@ export const Results = (props: ResultsProps): JSX.Element => {
       flowType === FLOWS.ATTACH &&
       currentStepIndex !== totalStepCount
     ) {
-      let axes: MotorAxes = mount === LEFT ? ['leftPlunger'] : ['rightPlunger']
-      // TODO: (sb)5/25/23 Stop homing leftZ for 96 once motor is disabled
-      if (attachedPipettes[mount]?.instrumentName === 'p1000_96') {
-        axes = ['leftPlunger', 'leftZ']
-      }
+      const axes: MotorAxes =
+        mount === LEFT ? ['leftPlunger'] : ['rightPlunger']
       chainRunCommands(
         [
           {
```

**Comment by sfoster1** [comment]
`getLabware.ts` resolution (add the flex tipracks to the excluded list):
```
@@ -57,6 +57,9 @@ export const LABWAREV2_DO_NOT_LIST = [
 export const PD_DO_NOT_LIST = [
   'opentrons_calibrationblock_short_side_left',
   'opentrons_calibrationblock_short_side_right',
+  'opentrons_flex_96_tiprack_200ul',
+  'opentrons_flex_96_tiprack_1000ul',
+  'opentrons_flex_96_tiprack_50ul',
 ]
 
 export function getLabwareV1Def(
```

**Comment by sfoster1** [comment]
- add the new labware

**Comment by sfoster1** [comment]
- Ignore the shared-data.patch changes

**Comment by sfoster1** [comment]
- Ignore the execute changes

File: api/src/opentrons/hardware_control/types.py [hunk 1]
```diff
@@ -190,7 +190,7 @@ def pipette_axes(cls) -> Tuple["OT3Axis", "OT3Axis"]:
     @classmethod
     def mount_axes(cls) -> Tuple["OT3Axis", "OT3Axis", "OT3Axis"]:
         """The axes which are used for moving instruments up and down."""
-        return cls.Z_L, cls.Z_R, cls.Z_G
```
Comment (sanni-t) [inline api/src/opentrons/hardware_control/types.py:193]: Was this an intentional change? As in, do we need to make sure we preserve this order always?

Comment (sfoster1) [reply api/src/opentrons/hardware_control/types.py:193]: Yeah I think so, or we need to replace the way we iterate through these in `home` with something else

File: api/release-notes-internal.md [hunk 2]
```diff
@@ -4,41 +4,30 @@ For more details about this release, please see the full [technical change log][
...
-- 96 channel attach flow fixes
-- Firmware updates now use an HTTP API; there's no frontend, so continue to follow the rules abotu restarting with an instrument attached after an update.
-- Several ODD white screen fixes
+- ODD should now display an error screen instead of a white screen
+- You can load labware to an OFF_DECK location
+- You can now use calibrate gripper in the reset settings options
+- There are consistent error codes in protocol status inforation now
+- The ODD should prompt you to update your pipettes when you attach one that needs it (!)
+- The 96 shouldn't fall down when you attach it now (!)
+- More little stall detection fixes
```
Comment (sanni-t) [inline api/release-notes-internal.md:22]: Sorry I forgot to add this note. Doesn't need to go in this PR if it's not the right thing to do. 
```suggestion
- More little stall detection fixes
- Randomized tip drop in trash is now changed to dropping tips in two predetermined locations in trash. This should fix any previous issues with pipettes crashing into the gantry when going to an inaccessible random tip drop location.
```

Comment (sfoster1) [reply api/release-notes-internal.md:22]: Yeah it is too late for that one unfortunately - this is just the mergeback

**Review by smb2268 — APPROVED** [review]
App-side changes involving gripper and pipette flows look good to me!

**Pull request merged**